pkg/apis/tensorflow/v1alpha2/defaults.go

```diff
@@ -88,7 +88,7 @@ func setTypeNameToCamelCase(tfJob *TFJob, typ TFReplicaType) {
 
 // SetDefaults_TFJob sets any unspecified values to defaults.
 func SetDefaults_TFJob(tfjob *TFJob) {
-	// Set default cleanpod policy to All.
+	// Set default cleanpod policy to running.
```

Comment:
s/running/Running

And https://github.com/jian-he/tf-operator/blob/0acc222f26ed60c14949af95eced7c22e5efd421/pkg/apis/tensorflow/v1alpha2/types.go#L48 also needs to be changed.